packages/grafana-e2e/src/flows/importDashboard.ts

```diff
@@ -0,0 +1,55 @@
+import { DeleteDashboardConfig } from '.';
+import { e2e } from '../index';
+import { fromBaseUrl, getDashboardUid } from '../support/url';
+
+type Panel = {
+  title: string;
+  [key: string]: any;
+};
+
+type Dashboard = { title: string; panels: Panel[]; [key: string]: any };
+
+/**
+ * Smoke test a datasource by quickly importing a test dashboard for it
+ * @param dashboardToImport a sample dashboard
+ */
+export const importDashboard = (dashboardToImport: Dashboard) => {
+  e2e().visit(fromBaseUrl('/dashboard/import'));
+
+  // Note: normally we'd use 'click' and then 'type' here, but the json object is so big that using 'val' is much faster
+  e2e.components.DashboardImportPage.textarea().invoke('val', JSON.stringify(dashboardToImport)).click({ force: true });
```

Comment:
So as a general rule, force: true, skips a few of the checks that a real user would have to do, ([docs here](https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Forcing)) so for example you can say click on a button without force true you have to first scroll down to where the button is. 

But this particular example that you've highlighted, *is* a little bit weird. I think for whatever reason using val sets the contents of the textarea but without clicking on the text area at some point, the load button will throw an error saying that you can't leave the textarea empty. I'm actually going to reverse this so that it's:
```
e2e.components.DashboardImportPage.textarea().click({ force: true }).invoke('val', JSON.stringify(dashboardToImport)).; 
```
I think it feels more like real life that you'd click on a textarea and then set the contents.